activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-model/pom.xml

```diff
@@ -118,6 +118,11 @@
       <artifactId>jackson-datatype-jsr310</artifactId>
       <version>2.16.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-jpamodelgen</artifactId>
```

Comment:
This dependency should enable the metamodel creation, but I don't see it being used in your changes. Is it a leftover?

Reply:
I miss the part where it was used, and I only see the one where it was missing 😄